decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super
```

Comment:
- [ ] The same happens here, please consider adding the parenthesis! 🤓 🔧 
```suggestion
    super()
```